oxygen-account/src/main/java/com/oxygenxml/account/messages/TokenClaims.java

```diff
@@ -0,0 +1,35 @@
+package com.oxygenxml.account.messages;
+
+import lombok.Getter;
+
+/**
+ * Defines constants for JWT token claims used in the application.
+ */
+@Getter
+public enum TokenClaims {
```

Comment:
Numele claselor de tip `enum` ar trebui sa fie la singular. Ideea e ca la un moment dat referiti o singura valoare dintr-un set, nu mai multe valori in acelasi timp. Valabil si pt celelalte enumuri.